### ISSUE TITLE
Is it possible to get a key in the old format?

### ISSUE THREAD

**Issue by AndriiiBut** [rzc0d3r/ESET-KeyGen#244]
Is it possible to get a key in the old format? Login + password.

**Comment by rzc0d3r** [comment]
No, but you can create an account (argument: --account), which will allow you a free period after authorization in the antivirus

**Comment by AndriiiBut** [comment]
And is it possible to get a login+password combination there? I didn't find such an option :(

**Comment by rzc0d3r** [comment]
If you mean with the EAV prefix, no way.

**Comment by rzc0d3r** [comment]
I meant that you can generate a regular account and then use it to activate the antivirus through it
